src/lwc/gsChecklist/gsChecklist.html

```diff
@@ -1,52 +1,31 @@
 <template>
     <article class="slds-setup-assistant__step">
         <div class="slds-summary-detail">
-            <button class="slds-button slds-button_icon slds-m-right_x-small slds-m-top_x-small"
+            <lightning-button-icon 
+                variant="container"
                 id={arrowId}
+                icon-name={arrowIcon}
+                class="slds-m-left_x-small"
                 onclick={onClickOpenOrClose}
                 aria-controls={expandedId}
                 aria-expanded={open}
                 title={group.title}>
```

Comment:
nice! I like using lightning-button-icon here. Looks like we lost the `alternative-text` though.